sintia/sintia.py

```diff
@@ -506,20 +506,30 @@ def format_quote(self, quote: Quote) -> str:
             'lbs': 'pounds',
             'kg': 'kilograms',
             'g': 'grams',
+            'm': 'meters',
+            'hPa': 'hectopascal',
+            'inHg': 'inch of mercury',
+            'inHg': 'inches of mercury',
+            'inHg': 'inhg',
+            'hPa': 'hpa',
+            'hPa': 'pascals',
```

Comment:
The mapping here is reversed. it's 

what is the alias: what it maps to

you know it's right if the unit on the right is what is in the conversion map on the bottom.